<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Syötä tähän oma nimesi&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nimi</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="weighingDateEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="Finnish" country="Finland"/>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mittauspäivä</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculatePushButton">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>730</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Laske</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savePushButton">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>730</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Tallenna</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="birthDateEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="Finnish" country="Finland"/>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1900</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Syntymäaika</string>
    </property>
   </widget>
   <widget class="QComboBox" name="genderComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="Finnish" country="Finland"/>
    </property>
    <property name="placeholderText" stdset="0">
     <string>Valitse sukupuoli</string>
    </property>
    <item>
     <property name="text">
      <string>Mies</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nainen</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sukupuoli</string>
    </property>
   </widget>
   <widget class="QFrame" name="dimensionFrame">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>20</y>
      <width>451</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/png/NaisSlice.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>360</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Rasvaprosentti (US)</string>
     </property>
    </widget>
    <widget class="QLabel" name="fatUsLabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>380</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Rasvaprosentti (FI)</string>
     </property>
    </widget>
    <widget class="QLabel" name="fatFiLabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Painoindeksi</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="bmiLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="sloganLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>341</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>UlosPallosta.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderNeck">
    <property name="geometry">
     <rect>
      <x>636</x>
      <y>230</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelNeck">
    <property name="geometry">
     <rect>
      <x>656</x>
      <y>200</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderWaist">
    <property name="geometry">
     <rect>
      <x>596</x>
      <y>460</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>30</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelWaist">
    <property name="geometry">
     <rect>
      <x>646</x>
      <y>420</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderHips">
    <property name="geometry">
     <rect>
      <x>566</x>
      <y>570</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelHips">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>50</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSliderHeight">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>49</y>
      <width>22</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
    <property name="value">
     <number>180</number>
    </property>
    <property name="sliderPosition">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QDial" name="dialWeight">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="minimum">
     <number>20</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>75</number>
    </property>
    <property name="sliderPosition">
     <number>75</number>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Paino</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pituus</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelWeight">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>75</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelHeight">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>55</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>180</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTiedosto">
    <property name="title">
     <string>Tiedosto</string>
    </property>
    <widget class="QMenu" name="menuJaa">
     <property name="title">
      <string>Jaa</string>
     </property>
     <addaction name="actionFacebook"/>
     <addaction name="actionTwitter"/>
     <addaction name="actionInstagram"/>
    </widget>
    <widget class="QMenu" name="menuAsetukset">
     <property name="title">
      <string>Asetukset</string>
     </property>
     <addaction name="actionKieli"/>
     <addaction name="actionUlkoasu"/>
    </widget>
    <addaction name="actionTulosta"/>
    <addaction name="menuJaa"/>
    <addaction name="menuAsetukset"/>
    <addaction name="actionSulje"/>
   </widget>
   <widget class="QMenu" name="menuMuokkaa">
    <property name="title">
     <string>Muokkaa</string>
    </property>
    <addaction name="actionPalauta_oletukset"/>
   </widget>
   <widget class="QMenu" name="menuOhjeet">
    <property name="title">
     <string>Ohjeet</string>
    </property>
    <addaction name="actionOhje"/>
   </widget>
   <addaction name="menuTiedosto"/>
   <addaction name="menuMuokkaa"/>
   <addaction name="menuOhjeet"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTulosta">
   <property name="text">
    <string>Tulosta...</string>
   </property>
  </action>
  <action name="actionSulje">
   <property name="text">
    <string>Sulje</string>
   </property>
  </action>
  <action name="actionFacebook">
   <property name="text">
    <string>Facebook</string>
   </property>
  </action>
  <action name="actionTwitter">
   <property name="text">
    <string>Twitter</string>
   </property>
  </action>
  <action name="actionInstagram">
   <property name="text">
    <string>Instagram</string>
   </property>
  </action>
  <action name="actionKieli">
   <property name="text">
    <string>Kieli</string>
   </property>
  </action>
  <action name="actionUlkoasu">
   <property name="text">
    <string>Ulkoasu</string>
   </property>
  </action>
  <action name="actionY_tila">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Yötila</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAutomaattinen_tallennus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automaattinen tallennus</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPoista_k_ytt_j">
   <property name="text">
    <string>Poista käyttäjä...</string>
   </property>
  </action>
  <action name="actionPalauta_oletukset">
   <property name="text">
    <string>Palauta oletukset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionOhje">
   <property name="text">
    <string>Ohje</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>birthDateEdit</tabstop>
  <tabstop>genderComboBox</tabstop>
  <tabstop>calculatePushButton</tabstop>
  <tabstop>savePushButton</tabstop>
  <tabstop>weighingDateEdit</tabstop>
 </tabstops>
 <resources>
  <include location="BGPictures.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSliderWaist</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelWaist</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>679</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderNeck</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelNeck</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>676</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderHips</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelHips</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderHeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelHeight</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialWeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelWeight</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
